src/controller/backend/Endpoints/UserManagementEndpoints.cs

```diff
@@ -0,0 +1,104 @@
+﻿using ApiSchema.Settings;
+using ApiSchema.Usermanagement;
+using Data.System;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backend.Endpoints;
+
+public static class UserManagementEndpoints
+{
+    public static void MapUserManagementEndpoints(this IEndpointRouteBuilder endpoints)
+    {
+        var group = endpoints.MapGroup("/api/usermanagement");
```

Comment:
Do we really want to type "usermanagement" every time? couldn't we just use "user" for example or are there any conflicts with existing endpoints?